src/bp/nlu/index.ts

```diff
@@ -6,14 +6,17 @@ import os from 'os'
 import path from 'path'
 
 import { NLUServerOptions } from './typings'
-export { NLUServerOptions as StanOptions } from './typings'
+export { NLUServerOptions } from './typings'
 
 const FILE_PATTERNS: Dic<RegExp> = {
   win32: /nlu-v(\d+_\d+_\d+)-win-x64\.exe/,
   darwin: /nlu-v(\d+_\d+_\d+)-darwin-x64/,
   linux: /nlu-v(\d+_\d+_\d+)-linux-x64/
 }
 
+global.DEBUG?.('nlu').sub('training')
+global.DEBUG?.('nlu').sub('prediction')
```

Comment:
What's the purpose of this? If it's only to display them to the user, please be specific. Also, you can create keys directly without calling sub. And DEBUG will never be undefined in this case, because it is executed by Botpress. So the ?. is not necessary.

Ex:
```js
// This will display those keys in the Debug panel so the user can enable them
DEBUG('nlu:training')
DEBUG('nlu:prediction')
```

Reply:
Not sure about an internal route for debug scopes. I feel like there are better solutions. This is just a quick fix that allow users to get back the previous behavior